_includes/header.html

```diff
@@ -3,7 +3,6 @@
     <meta content='width=device-width, initial-scale=1.0' name='viewport' />
     <meta content='Clojure graph library built on top of Aurelius Titan and the Tinkerpop stack' name='description' />
     <meta content='The ClojureWerkz Team' name='author' />
-    <link rel="icon" type="image/png"  href="//www.synapse.org/favicon.png?v=1" />
```

Comment:
It looks like all of these values specified in the <head> of header.html end up in the <body> of the output html page.